EventsExpress.Core/IServices/IBaseService.cs

```diff
@@ -1,19 +1,17 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using System.Text;
 
-namespace EventsExpress.Db.IRepo
+namespace EventsExpress.Db.IBaseService
 {
-    public interface IRepository<T>
+    public interface IBaseService<T>
```

Comment:
If we are making this interface, please move it to services project. Do not keep it in DB.